### PR TITLE
Test folder

### PR DESCRIPTION
**Pull request by victuxbb**
* Move test package inside source code folder

### CODE REVIEW

**Comment by victuxbb** [comment]
Yes, we will have "src/test/cmp" and that's the idea and in this project is the perfect example of why is this a good option.
* src folder means "SOURCE CODE" and tests are source code, so should be inside src folder.
* Inside src you can have more than one "app" in this project for example, we have _cmp_ and _globalstore_ so you will have source code testing for _cmp_ and _globalstore_ right?

For example if you pick up any random project of our company in general you will find this structure forlder: https://github.schibsted.io/scmspain/ms-common--locations/tree/master/src
And you can see that there is a test folder and inside is the repeated app package.

In summary, you should ask yourself, are tests source code?

BTW, OpenAds is already using this structure ;) https://github.com/scm-spain/OpenAds/tree/master/src
